test/fixtures/identify-location.json

```diff
@@ -0,0 +1,49 @@
+{
+  "input": {
+    "type": "identify",
+    "userId": "user-id",
+    "timestamp": "2014",
+    "traits": {
+      "created": "2014-01-01",
+      "email": "jd@example.com",
+      "firstName": "john",
+      "lastName": "doe"
+    },
+    "context": {
+      "location": {
+        "city": "Dublin",
+        "country": "Ireland",
```

Comment:
What happens in the case where the user passes a country that isn't title cased?